linode/manager
# Remove unused `babel` dependencies

## Description 📝

- Removes Babel because it is no longer needed
  - We no longer depend on Babel to build Cloud Manager
  - We no longer need Babel to transform for unit testing

## How to test 🧪

- Test local development `yarn dev` and ensure it still works as expected
- Test building Cloud Manager for production `yarn build` and serving it with `yarn start:ci`
- Test Storybook development with `yarn storybook`
- Test building Storybook with `yarn build-storybook`
